client/app/components/TagsList.less

```diff
@@ -13,7 +13,8 @@
       margin: 0;
     }
 
-    a {
+    a,
+    button {
```

Comment:
This is a pretty broad change -- it might be right, but I want to confirm: in this context, will there ever be `button`'s that we don't want to match this style (e.g. `button`'s that aren't styled like links, such as antd's `Button`)? If so, just switch `button` to some classname and we're all good.

Reply:
(This also happens [here](https://github.com/getredash/redash/pull/5420/files?w=1#diff-05d6220f9633063267ad9eb66926b5847c1461e1da03108f3ff54f4610dac030R255-R256))

Reply:
It shouldn't become a problem, but I agree we're safer having a class.